tilecloud_chain/controller.py

```diff
@@ -137,19 +137,21 @@ def _get(path, cache):
             return file.read()
 
 
-def _validate_generate_wmts_capabilities(cache):
+def _validate_generate_wmts_capabilities(cache, exit):
     if 'http_url' not in cache and 'http_urls' not in cache:  # pragma: no cover
         logger.error(
             "The attribute 'http_url' or 'http_urls' is required in the object cache[{}].".format(
                 cache['name']
             )
         )
-        exit(1)
+        if exit:
+            exit(1)
```

Comment:
I didn't know that True was a callable?!?!?

Reply:
:-)